gtk3: 3.24.31 -> 3.24.33

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for gtk3 is: "A multi-platform toolkit for creating graphical user interfaces"

meta.homepage for gtk3 is: "https://www.gtk.org/"

meta.changelog for gtk3 is: "https://gitlab.gnome.org/GNOME/gtk/-/raw/3.24.33/NEWS"


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- Warning: no invocation of /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33/bin/gtk-launch had a zero exit code or showed the expected version
- Warning: no invocation of /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33/bin/gtk-update-icon-cache had a zero exit code or showed the expected version
- Warning: no invocation of /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33/bin/broadwayd had a zero exit code or showed the expected version
- 0 of 3 passed binary check by having a zero exit code.
- 0 of 3 passed binary check by having the new version present in output.
- found 3.24.33 with grep in /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33
- found 3.24.33 in filename of file in /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
6739 total rebuild path(s)

6160 package rebuild(s)

6158 x86_64-linux rebuild(s)
581 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
AusweisApp2
CHOWTapeModel
ChowKick
CuboCore.coreaction
CuboCore.corearchiver
CuboCore.corefm
CuboCore.coregarage
CuboCore.corehunt
CuboCore.coreimage
CuboCore.coreinfo
CuboCore.corekeyboard
CuboCore.corepad
CuboCore.corepaint
CuboCore.corepdf
CuboCore.corepins
CuboCore.corerenamer
CuboCore.coreshot
CuboCore.corestats
CuboCore.corestuff
CuboCore.coreterminal
CuboCore.coretime
CuboCore.coretoppings
CuboCore.coreuniverse
CuboCore.libcprime
CuboCore.libcsys
DisnixWebService
HentaiAtHome
MIDIVisualizer
OSCAR
PageEdit
R
Sylk
_1password-gui
abcl
abiword
adapta-gtk-theme
adementary-theme
adoptopenjdk-bin
adoptopenjdk-hotspot-bin-11
adoptopenjdk-hotspot-bin-13
adoptopenjdk-hotspot-bin-14
adoptopenjdk-hotspot-bin-15
adoptopenjdk-hotspot-bin-16
adoptopenjdk-hotspot-bin-8
adoptopenjdk-icedtea-web
adoptopenjdk-jre-bin
adoptopenjdk-jre-hotspot-bin-11
adoptopenjdk-jre-hotspot-bin-13
adoptopenjdk-jre-hotspot-bin-14
adoptopenjdk-jre-hotspot-bin-15
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A gtk3 https://github.com/r-ryantm/nixpkgs/archive/f34237d084625b2c4c4d4dbae61107c182894d76.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33
ls -la /nix/store/r2hk3r975i44jh8k6xfazq4nybckiwsm-gtk+3-3.24.33/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @7c6f434c @hedning @jtojnar @dasj19 @maxeaubrey for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jtojnar reviewed (approved):
(https://ftp.gnome.org/pub/GNOME/sources/gtk+/3.24/gtk+-3.24.32.news)

> Overview of Changes in GTK+ 3.24.32, 03-03-2022
> ===============================================
> 
> * GtkCellRendererProgress:
>   - Use tabular figures
> 
> * GtkFontChooser:
>   - Fix the build with older Pango
>   - Fix axis name handling
> 
> * Theme:
>   - Fix border color for tiled windows
> 
> * Accessibility:
>   - Fix cell accessible leak
> 
> * Wayland:
>   - Support new high-contrast setting
>   - Only update scale when on any outputs
> 
> * Windows:
>   - Various build fixes
>   - Fix keyboard state handling
>   - Fix capslock handling
>   - Fix AltGr not working
> 
> * MacOS:
>   - Fix the build on 10.11
>   - Make Dock "Quit" invoke the app.quit action
>   - Correctly handle HiDPI cursors
> 
> * Translation updates



(https://ftp.gnome.org/pub/GNOME/sources/gtk+/3.24/gtk+-3.24.33.news)

> Overview of Changes in GTK+ 3.24.33, 04-03-2022
> ===============================================
> 
> * No changes

jtojnar committed:
gtk3: 3.24.31 → 3.24.33

https://ftp.gnome.org/pub/GNOME/sources/gtk+/3.24/gtk+-3.24.32.news
https://ftp.gnome.org/pub/GNOME/sources/gtk+/3.24/gtk+-3.24.33.news

jtojnar commented:
Darwin failure looks unrelated.
